cloudformation/membership-attribute-service.yaml

```diff
@@ -80,6 +86,7 @@ Mappings:
       DynamoDBBehaviourTable: arn:aws:dynamodb:*:*:table/MembershipBehaviour-UAT
       DynamoDBBehaviourTableTestUsers: arn:aws:dynamodb:*:*:table/MembershipBehaviour-UAT
       GiraffeTopic: giraffe-code
+      SSLCertificate: arn:aws:iam::865473395570:server-certificate/members-data-api.theguardian.com
```

Comment:
this is just a duplicate of the above ssl cert, if we really want to use this properly we should register members-data-api.code.dev-etc and get a cert for that